tests/check_framework/test_security.py

```diff
@@ -471,3 +472,35 @@ def test_with_referrer_policy(self):
     )
     def test_with_invalid_referrer_policy(self):
         self.assertEqual(base.check_referrer_policy(None), [base.E023])
+
+
+def failure_view_with_invalid_signature():
+    pass
+
+
+class CSRFFailureViewTest(SimpleTestCase):
+    @override_settings(CSRF_FAILURE_VIEW='')
+    def test_failure_view_import_error(self):
+        self.assertEqual(
+            csrf.check_csrf_failure_view(None),
+            [
+                Error(
+                    "The CSRF failure view '' could not be imported.",
+                    id='security.E025',
+                )
+            ],
+        )
+
+    @override_settings(
+        CSRF_FAILURE_VIEW='%s.failure_view_with_invalid_signature' % __name__,
```

Comment:
why not an f-string as per my suggestion? We're on Python 3.6 only now.

Reply:
replaced with  `f-string`